### PR TITLE
clear old bootstrap files

### PR DESCRIPTION
**Pull request by jack-han-115**
Fixed issue #<Mantis issue number>:
New feature #<Mantis issue number>:
Related to: [previous PR](https://github.com/LimeSurvey/LimeSurvey/pull/3131)

### CODE REVIEW

File: application/core/QuestionTypes/ArrayFlexibleRow/RenderArrayFlexibleRow.php [hunk 1]
```diff
@@ -290,20 +287,19 @@ public function getNonDropdownRows()
                     'ld' => $oAnswer->code,
                     'code' => $oAnswer->code,
                     'label' => $oAnswer->answerl10ns[$this->sLanguage]->answer,
-                    'checked' => ($this->getFromSurveySession($myfname) == $oAnswer->code) ? 'checked' : '',
```
Comment (olleharstedt) [inline application/core/QuestionTypes/ArrayFlexibleRow/RenderArrayFlexibleRow.php:293]: Is this correct? Seems unrelated to PR title?

Comment (jack-han-115) [reply application/core/QuestionTypes/ArrayFlexibleRow/RenderArrayFlexibleRow.php:293]: https://github.com/LimeSurvey/LimeSurvey/pull/3189

Comment (jack-han-115) [reply application/core/QuestionTypes/ArrayFlexibleRow/RenderArrayFlexibleRow.php:293]: I updated PR link in the Jira ticket.
